board/piksiv3/rootfs-overlay/etc/init.d/S85piksi_firmware

```diff
@@ -50,8 +56,16 @@ services_up()
 
 wait_services()
 {
+  local uart_wait_count=0
+
+
   while ! services_up; do
-    sleep 0.1;
+    if [[ $uart_wait_count -ge $uart_wait_count_limit ]]; then
+      loge --sbp "Timeout expired while waiting for UARTs"
+      break;
+    fi
+     sleep $uart_check_interval
+    uart_wait_count=$(($uart_wait_count + 1))
```

Comment:
Please fix indent here, our shell coding style (for piksi_buildroot) is two spaces for indentation.